src/node/desktop/src/main/product-info.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * product-info.ts
+ *
+ * Copyright (C) 2021 by RStudio, PBC
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+export function getRStudioVersion() {
+  // TODO: need to figure out how to consume build numbers for package build, etc.
```

Comment:
We could probably have something like `product-info.ts.in` and generate `product-info.ts` from that in CMake.

Reply:
Need to learn more about versioning at the Electron API level. For example, stumbled across this, not sure what role is plays or where...

https://www.electronjs.org/docs/api/structures/product